### DISCUSSION TITLE
Help with those buttons on Repositories

### DISCUSSION THREAD

**Discussion by byemc** [community/community#23194] [New to GitHub]
<p>You know those “Works on Python | 3.8 | 3.9” or “Main Discord | 1000 online” buttons? How to I add them to my own repo?</p>

**Comment by kingthorin** [comment, marked as answer]
<p>Checkout <a href="http://shields.io" rel="noopener nofollow ugc">shields.io</a> for badges, you just add them with standard markdown image tags.</p>

**Comment by byemc** [comment]
<p>Thank you very much.</p>
